astropy/cosmology/_src/funcs/optimize.py

```diff
@@ -17,13 +25,16 @@
 # isort: split
 from astropy.cosmology import units as cu
 from astropy.cosmology._src.core import CosmologyError
+from astropy.cosmology._src.scipy_compat import minimize_scalar
 from astropy.cosmology._src.typing import FArray
 
 if TYPE_CHECKING:
     import scipy.optimize
 
 __doctest_requires__ = {"*": ["scipy"]}
 
+METHOD_UNBOUNDED: Final = frozenset({"brent", "golden"})
```

Comment:
Why not implement a type alias instead?

Reply:
Maybe I'm not understanding... this is for a runtime `is in` check.